data-source/jsonnet/northern-ireland/blocks/household/accommodation/accommodation_type.jsonnet

```diff
@@ -0,0 +1,56 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import '../../../lib/rules.libsonnet';
+
+{
+  type: 'Question',
+  id: 'accommodation-type',
+  question: {
+    id: 'accommodation-type-question',
+    title: {
+      text: 'What type of accommodation is <em>{address}</em>?',
+      placeholders: [placeholders.address],
+    },
+    type: 'General',
+    answers: [
+      {
+        id: 'accommodation-type-answer',
+        mandatory: false,
+        type: 'Radio',
+        options: [
+          {
+            label: 'Whole house or bungalow',
+            value: 'Whole house or bungalow',
+          },
+          {
+            label: 'Flat, maisonette or apartment',
+            value: 'Flat, maisonette or apartment',
+            description: 'Including purpose-built flats and flats within converted and shared houses',
+          },
+          {
+            label: 'Caravan or other mobile or temporary structure',
+            value: 'Caravan or other mobile or temporary structure',
+          },
+        ],
+      },
+    ],
+  },
+  routing_rules: [
+    {
+      goto: {
+        block: 'type-of-house',
+        when: [rules.accommodationIsHouse],
+      },
+    },
+    {
+      goto: {
+        block: 'type-of-flat',
+        when: [rules.accommodationIsFlat],
+      },
+    },
+    {
+      goto: {
+        block: 'adapted',
```

Comment:
Should this be the same as the eng & wal or different, as it currently different for the bare goto

Reply:
The goto in the England & Wales schema is 'self-contained', just want to check this is a known difference

Reply:
https://github.com/ONSdigital/eq-survey-runner/blob/29bfb158cb027aa6be209ca484338ff2673c5a74/data-source/jsonnet/england-wales/blocks/household/accommodation/accommodation_type.jsonnet#L52

Reply:
Yeah this is supposed to have changed, the 'self-contained' question has been removed completely, and the 'adapted' question is a new one for NI.